content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -5,130 +5,139 @@ Troubleshooting
+the printer in the list. In the Administration dropdown, click on :guilabel:`Modify Printer`.
+Follow the steps and select the :guilabel:`Make and Model` corresponding to the printer.
 
 .. image:: troubleshooting/troubleshooting_printer_02.png
    :align: center
+   :alt: Edit the printer connected to the IoT box.
 
 .. note::
-   Epson and Star receipt printers and Zebra label printers do not
-   need a driver to work. Make sure that no driver is selected for those
-   printers.
+   Epson and Star receipt printers and Zebra label printers do not need a driver to work. Make sure
+   that no driver is selected for those printers.
 
-My Zebra Printer doesn't print anything
----------------------------------------
+The Zebra Printer doesn't print anything
+----------------------------------------
 
-Zebra printers are quite sensitive to the format of the ZPL code that is
-printed. If nothing comes out of the printer or blank labels are printed,
-try changing the format of the report that is sent to the printer by
-accessing :menuselection:`Settings ~~> Technical ~~> Views` in developer
-mode and look for the corresponding template.
+Zebra printers are quite sensitive to the format of the ZPL code that is printed. If nothing comes
+out of the printer or blank labels are printed, try changing the format of the report that is sent
+to the printer by accessing :menuselection:`Settings --> Technical --> Views` in developer mode and
+look for the corresponding template.
 
 Barcode Scanner
 ===============
 
 The characters read by the barcode scanner don't match the barcode
 ------------------------------------------------------------------
 
-By default, we assume that your barcode scanner is configured in US
-QWERTY. This is the default configuration of most barcode readers.
-If your barcode scanner uses a different layout, please go to the
-form view of your device and select the correct one.
+By default, most barcode scanners are configured in the US QWERTY format. If the barcode scanner
+uses a different layout, go to the form view of the device and select the correct one.
 
 Nothing happens when a barcode is scanned
 -----------------------------------------
 
-Make sure that the correct device is selected in your Point of Sale
-configuration and that your barcode is configured to send an ENTER
-character (keycode 28) at the end of every barcode.
+Make sure that the correct device is selected in the Point of Sale configuration and that the
+barcode is configured to send an ENTER character (keycode 28) at the end of every barcode.
 
 The barcode scanner is detected as a keyboard
 ---------------------------------------------
 
-Some poorly built barcode scanners do not advertise themselves as
-barcode scanners but as a USB keyboard instead, and will not be
-recognized by the IoT Box.
+Some poorly built barcode scanners do not advertise themselves as barcode scanners but as a USB
```

Comment:
```suggestion
Some barcode scanners do not advertise themselves as barcode scanners but as a USB
```

Reply:
This might be worth a warning or a note

Reply:
I've added this in as a important note. Thanks for suggesting new wording on this sentence.